src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
```

Comment:
>NuGetTargetMoniker [](start = 38, length = 18)

Can you change this condition to be `'$(TargetGroup)'==''`. I know that this would also work, but we have been trying to keep most conditions in csprojs to be about TargetGroup or OSGroup to make it easier to understand the relationship between .builds and .csproj